docs/cn/ufs/Azure-Data-Lake.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: global
+title: Azure 数据湖存储
+nickname: Azure 数据湖存储
+group: Storage Integrations
+priority: 2
+---
+
+* Table of Contents
+{:toc}
+
+本指南介绍如何配置Alluxio，使其与底层存储系统 [Azure Data Lake Storage Gen1](https://docs.microsoft.com/en-in/azure/data-lake-store/data-lake-store-overview) 一起运行。 
```

Comment:
how about linking to the Chinese doc for Azure Data Lake https://learn.microsoft.com/zh-cn/azure/data-lake-store/data-lake-store-overview